pkg/reconciler/contour/config/contour.go

```diff
@@ -31,14 +31,16 @@ const (
 	// customizations for contour related features.
 	ContourConfigName = "config-contour"
 
-	visibilityConfigKey = "visibility"
+	visibilityConfigKey       = "visibility"
+	defaultTLSSecretConfigKey = "default-tls-secret-name"
 )
 
 // Contour contains contour related configuration defined in the
 // contour config map.
 type Contour struct {
-	VisibilityKeys    map[v1alpha1.IngressVisibility]sets.String
-	VisibilityClasses map[v1alpha1.IngressVisibility]string
+	VisibilityKeys       map[v1alpha1.IngressVisibility]sets.String
+	VisibilityClasses    map[v1alpha1.IngressVisibility]string
+	DefaultTLSSecretName string
```

Comment:
`types.NamespacedName`?

Reply:
Yeah we should fully qualify it with the namespace